docs/topics/Permissions.md

```diff
@@ -48,6 +48,14 @@ Permissions follow a hierarchy with the following roles:
 * Bots can only sort roles lower than their highest role.
 * Bots can only kick/ban users of with a lower highest role than themselves.
 
+In the case of permissions overlapping with each other—e.g. a user's role is banned from seeing a channel, but the user has a specific override—they are applied to a user in the following order:
```

Comment:
this should really be in a separate heading called `## Permission Calculation` since hierarchy is just how your permissions on the guild respect the order of the roles list.

nit: would prefer moving the example to the end of the sentence, maybe wrapping in parens instead